src/cpp/src/visual_language/inputs_embedder.cpp

```diff
@@ -218,6 +219,22 @@ std::pair<ov::Tensor, ov::Tensor> InputsEmbedder::IInputsEmbedder::get_inputs_em
     OPENVINO_THROW("This model does not support token_type_ids.");
 }
 
+std::pair<ov::Tensor, ov::Tensor> InputsEmbedder::IInputsEmbedder::get_inputs_embeds_with_token_type_ids(
+    const std::string& prompt,
+    const std::vector<EncodedImage>& images,
+    const std::vector<ov::genai::EncodedVideo>& videos,
+    VLMPerfMetrics& metrics,
+    bool recalculate_merged_embeddings,
+    const std::vector<size_t>& image_sequence,
+    const std::vector<size_t>& videos_sequence,
+    const std::vector<std::pair<std::size_t, std::size_t>> history_vision_count) {
+    if (videos.size() > 0) {
+        OPENVINO_THROW("The model doesn't support 'videos' preprocessing yet. Please use 'images' instead.");
```

Comment:
please use `OPENVINO_ASSERT` with condition `videos.size()==0` No need in if-else.

Reply:
Updated.